hyperspy/learn/mva.py

```diff
@@ -958,6 +959,170 @@ def plot_explained_variance_ratio(self, n=50, log=True):
             ax.semilogy()
         return ax
 
+    def scree_plot(self, n=50, log=True, cutoff=0.01, signal_num=0,
```

Comment:
I wouldn't change the name of the function. This is not exactly the scree plot as it doesn't plot the eigenvalues but the explained variance ratio. (And not even that when using poisson noise normalization...) So, why not simply replacing the old function?